stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MessageReplyView.kt

```diff
@@ -120,16 +123,16 @@ internal class MessageReplyView : FrameLayout {
     /**
      * Sets the background for message reply.
      *
-     * @param message [Message] The message containing reply.
-     * @param isMine Whether the message is from the current user or not.
+     * @param message [Message] The message replied message.
+     * @param isMine Whether the message containing the reply is from the current user or not.
      * @param style [MessageReplyStyle] contains the styles of the background.
      */
     private fun setReplyBackground(message: Message, isMine: Boolean, style: MessageReplyStyle?) {
         val shapeAppearanceModel = ShapeAppearanceModelFactory.create(
             context,
             REPLY_CORNER_RADIUS,
             0f,
-            isMine,
+            message.isMine(),
```

Comment:
same here

Reply:
Same reasoning as above, just for the shape.

Reply:
Sounds good - my bad!